pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -69,62 +88,78 @@ In plain language, this code says:
+  After you load the USWDS source code, you can build new styles with USWDS design tokens, functions, and mixins. For the purposes of this guide, we won’t get into custom code, but the important thing to understand is that any custom code should follow the settings and USWDS source code in your Sass entry point.
+
+## Using uswds-compile
+Complete the following steps to install, configure, and use `uswds-compile`:
+
+### Step 1: Install uswds-compile
 
-## Step 2: Install uswds-compile
 From your project’s root, run the following command to install `uswds-compile`:
 
-{:.site-terminal}
 ```bash
 npm install @uswds/compile --save-dev
 ```
 
-## Step 3: Create a gulpfile
-Create a file called `gulpfile.js` at the root of your project by running the command `touch gulpfile.js` (alternatively, use an existing gulpfile if one already exists). This file needs to do the following:
-- import the `@uswds/compile` package
-- set any project settings
-- export the functions and/or tasks you need
+### Step 2: Create a gulpfile
 
-Your `gulpfile.js` may read as follows:
+Once installed, create a file called `gulpfile.js` at the root of your project by running the command `touch gulpfile.js` (alternatively, use an existing gulpfile if one already exists). This file needs to do the following:
 
-  {:.site-terminal}
-  ```scss
+- Import the `@uswds/compile` package
+- Set the project's USWDS version
+- Set custom project path settings
+- Export the Gulp functions and/or tasks you need
+
+You can find details about configuring your `gulpfile.js` in steps 3-5.
+
+For an initial example, a simple `gulpfile.js` may read as follows:
+
+```scss
 /* gulpfile.js */
 
+/**
+* Import uswds-compile
+*/
 const uswds = require("@uswds/compile");
 
 /**
- * USWDS version
- * Set the version of USWDS you're using (2 or 3)
- */
-
+* USWDS version
+* Set the major version of USWDS you're using
+* (Current options are the numbers 2 or 3)
+*/
 uswds.settings.version = 3;
 
 /**
- * Path settings
- * Set as many as you need
- */
-
+* Path settings
+* Set as many as you need
+*/
 uswds.paths.dist.css = './assets/css';
 uswds.paths.dist.sass = './sass';
 
 /**
- * Exports
- * Add as many as you need
- */
-
+* Exports
+* Add as many as you need
+*/
 exports.init = uswds.init;
 exports.compile = uswds.compile;
-  ```
+exports.watch = uswds.watch;
+```
+
+### Step 3: Set USWDS version
+
+Once your `gulpfile.js` is created, declare which major version of USWDS your project uses with `settings.version`. This configuration tells Gulp where to find USWDS source files in the `node_modules` directory. This setting currently accepts values of the numbers `2` and `3` to represent USWDS 2.X and 3.X, respectively.
 
-## Step 4: Create path settings and export compile functions
-Now, create your path settings using the following table:
+### Step 4: Customize path settings
```

Comment:
Added detail to explain what the path settings are for and examples that show how a user might want to customize